marklogic-data-hub-central/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.test.tsx

```diff
@@ -37,7 +38,225 @@ describe('RTL Source-to-entity map tests', () => {
         fireEvent.click(getByText('Clear'));
         expect(getByText('Clear')).toBeEnabled();
         expect(getByText("concat(name,'-NEW')")).toBeInTheDocument();
-        console.log(container);
+    });
+
+    test('Filtering Name column in Source and Entity tables',() => {
+      
+      const { getByText,getByTestId,queryByText } = render(<SourceToEntityMap {...data.mapProps} 
+        mappingVisible={true} 
+        entityTypeProperties={data.mapProps.entityTypePropertiesUpdated} 
+        sourceData={data.mapProps.sourceDataUpdated}/>);
```

Comment:
we can create and store json and xml source data outside of mapProps and default mapProps.sourceData to use json. This way we can simply say sourceData={data.xmlSourceData}. In other words all the overrides can stay outside of mapProps.

Reply:
Done.